tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -48,8 +48,6 @@ def test_streams_resilient_to_broker_down(self):
         processor = StreamsBrokerDownResilienceService(self.test_context, self.kafka, self.get_configs())
         processor.start()
 
-        # until KIP-91 is merged we'll only send 5 messages to assert Kafka Streams is running before taking the broker down
-        # After KIP-91 is merged we'll continue to send messages the duration of the test
```

Comment:
Cleanup: TODO comments are not an effective project planning mechanism. If there's something that needs to be done, we should create a Jira. But really, it just sounds like a slightly different alternative implementation of the same test, which we may or may not want to do. Thus, I'm just deleting it.